### DISCUSSION TITLE
Misorientation calculation considering crystal structure symmetries

### DISCUSSION THREAD

**Discussion by mauroarcidiacono** [mtex-toolbox/mtex#1428] [Ask Anything] [answered]
Hi,

I want to calculate the misorientation between grains considering the symmetries due to the crystal structure. As input, I have the Euler angles. I tried using calcMisorientation but I could not find a way to perform the calculation. I want something that looks like this:

Function ([‘cubic’, Euler Angles 1], [‘hexagonal’, Euler Angles 2]) = misorientation

It does not have to be exactly that expression, but one or a set of expressions that will allow me to obtain the misorientation. Is it possible to do this with MTEX?

Thank you very much for your help.

**Comment by kilir** [comment, marked as answer]
Hi,
try
```
csa = crystalSymmetry('432','mineral','a')
csb = crystalSymmetry('321','mineral','b')

oria = orientation.byEuler([10 11 12]*degree,csa)
orib = orientation.byEuler([13 14 15]*degree,csb)

orib\oria
% or 
inv(orib) * oria

```
Cheers,
Rüdiger

**Reply by mauroarcidiacono** [reply]
Hi,

Thank you very much Rüdiger, the code you wrote solved my problem. For those who have a similar question, I used the following expression to calculate the misorientation angle from oria and orib:

`m = angle(oria, orib) ./ degree;`

Kind regards,
Mauro Francisco Arcidiacono
